Libraries/Components/TVView/TVView.ios.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule TVView
+ */
+'use strict';
+
+const React = require('React');
+const View = require('View');
+const ReactNativeViewAttributes = require('ReactNativeViewAttributes');
+const StyleSheetPropType = require('StyleSheetPropType');
+const ViewStylePropTypes = require('ViewStylePropTypes');
+
+const requireNativeComponent = require('requireNativeComponent');
+
+const PropTypes = React.PropTypes;
+
+/**
+ * Extension of the View component that supports the Apple TV focus engine.
+ * Used by Touchable and Navigation components to support user input using the Apple TV remote.
+ *
+ * @extends View
+ * @platform ios
+ */
+class TVView extends View {
```

Comment:
Why does this need to extend View? I'm not sure if that's supported (if it works though...)